File: Back-End/buyer_apps/src/main/java/portfolio/online/shop/buyer_apps/Models/Product.java
package portfolio.online.shop.buyer_apps.Models;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Table(name="Products")
@Entity
public class Product {
    @Id
    @Column(name="product_id")
    private String product_id;

    @Column(name="user_id")
    private String user_id;

    @Column(name="product_name")
    private String product_name;

    @Column(name="product_price")
    private Double product_price;

    @Column(name="product_description")
    private String product_description;

    @Column(name="created_date")
    private String created_date;

    @Column(name="modified_datetime")
    private String modified_datetime;
}

File: Back-End/seller_apps/Models/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;
using seller_apps.DTOs;

namespace seller_apps.Models
{
    [Table("Products")]
    public class Product
    {
        [Key]
        public string product_id {get; set;} = String.Empty;
        public string user_id {get; set;} = String.Empty;
        public string product_name {get; set;} = String.Empty;
        public decimal product_price {get; set;}
        public string? product_description {get; set;}
        public DateTime created_date {get; set;}
        public DateTime? modified_datetime {get; set;}

        // public static Product ConvertDtoToObject(ProductDTO productDTO){
        //     return new()
        //     {
        //         product_name = productDTO.product_name,
        //         product_price = productDTO.product_price,
        //         product_description = productDTO.product_description
        //     };
        // }
    }
}